pythosf/client.py

```diff
@@ -60,22 +63,22 @@ def json_api_request(self, url, method=None, item_id=None, item_type=None, attri
 
         while keep_trying:
             keep_trying = False
+            combined_headers = combine_headers(self.base_headers(token=token), headers)
```

Comment:
Is this going to save in a Jupyter notebook? I had it in the actual request to keep from storing it into a variable.

Reply:
Ah okay. Makes sense. Will change back! (Keep forgetting about that!)

Reply:
Yeah, it's weird. I should just give it up, really. But I am stubborn so far.